### ISSUE TITLE
Error updating API in a non default namespace

### ISSUE THREAD

**Issue by renuka-fernando** [wso2/product-apim-tooling#473]
**Description:**
Error updating API in a non default namespace: `wso2-demo`
```sh
$ apictl update api -n online-store --from-file=./swagger.yaml --namespace=wso2-demo

Output:
apictl: Could not find the API with the name "online-store"
Exit status 1
```

**Affected Product Version:**
apictl 3.2.0-RC2  

**Steps to reproduce:**
1. Add API in `wso2-demo` namespace
1. Update API same API in `wso2-demo` namespace

**Comment by renuka-fernando** [comment]
Workaround for this issue

```sh
$ apictl config set-context --current --namespace=wso2-demo
```

**Comment by npamudika** [comment]
This issue is fixed in APICTL 3.2.1 release - https://github.com/wso2/product-apim-tooling/releases/tag/v3.2.1
